openseeds/cohort-procedures-and-templates/graduation-guide.md

```diff
@@ -0,0 +1,134 @@
+
+## These are the requirements to be able to graduate from the Open Seed programme by OLS:
+
+1. Attend at least 5 mentor-mentee meetings, unless there is a valid reason their mentors can confirm (The Open Seeds program includes 8 mentor-mentee meetings in total).
+2. A “yes” from your mentor(s) confirming your participation and engagement in OLS curriculum.
+3. Open Canvas discussed with their mentor.
+4. Vision and Mission statements discussed with their mentor.
+5. Roadmap discussed with their mentor.
+6. Attend cohort calls, or watch recorded videos.
+7. Participate in at least one skill up session, q&a, or thematic session, unless there is a valid reason their mentors can confirm.
+8. Complete the mid term survey for project leads/mentees.
+9. Present a 5 minute presentation in one of the graduation calls, or send a recording of a 15 minutes presentation created exclusively for Open Seeds OLS-8. 
+
+## Types of graduation
+Participants can graduate in two different ways: 
+- Live presentation: by registering and then joining one of the three graduation sessions and presenting their project for up to 5 minutes (we are strict with time)
+- Pre-recorded presentation: by recording a 15 minute presentation and sending it to the OLS team by the end of week 16. These recordings will later be uploaded to our YouTube channel, and will also be shared on the ols-8 Slack channel (and other channels), for other participants to see. 
+
+## Preparing for graduation 
+Do remember to sign up for and attend one of the graduation rehearsals, so you can get feedback and ideas for your presentation. For OLS-8, check the times in the schedule or public calendar. 
+
+## What we expect from you at the graduation
+- We expect you to discuss your plan with your mentor in the meeting before graduation, prepare your presentation and be ready to answer questions from fellow participants, live during graduation, or via slack if you pre-record your graduation.
+- We want to hear about your project, the progress you have made, the hurdles you encountered, and what comes next. If you want, use the chance to invite others to join you as collaborators or advisers.
+- If possible, please use the Open Seeds slides template and OLS logos in your presentation.
+- If it is a group project, we would prefer if all project members are present during the graduation call, and more than one person speaks/delivers the presentation.
+- Please upload your presentation on Zenodo, and include 'openlifesci' in the 'community' section. This will allow us to feature your presentation on our Zenodo community page. 
+
+## What happens if an emergency occurs? 
+If something unexpected happens and you cannot prepare your presentation in the expected timeframe, please get in touch with us (team@we-are-ols.org) and we will get back to you to discuss your case. 
```

Comment:
Please use Jekyll variable for the email

Reply:
Done. Thanks!